dev/2018/spring/knights-to-kings/knights-to-kings.html

```diff
@@ -0,0 +1,196 @@
+<article class="story story-default">
+  <div class="container">
+    <div class="row">
+        <img src="[image filename='Knights-to-Kings-UCF-Football']" alt="From Knights to Kings" title="From Knights to Kings" class="img-responsive center-block story-header">
+        <h1 class="sr-only">From Knights to Kings: UCF Football players reflect on the 2017 season</h1>
+    </div>
+  </div>
+  <div class="story-deck" [background-image filename="Gold-BG"]>
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-10 col-xs-offset-1 text-center">
+          <h2>Inside the team that went from<br class="hidden-sm hidden-xs"> zero wins to National Champions.</h2>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container mt-5">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1">
+        <div class="row">
+          <div class="col-sm-7">
+              <p class="byline">By <strong>Eric Michael &rsquo;96</strong></p>
+            </div>
+            <div class="col-sm-5">
+              <div class="social-wrap">[social_buttons]</div>
+            </div>
+        </div>
+        <div class="row">
+          <div class="col-md-12 story-dropcap" [background-image filename='Knights-Dropcap']>
+            <p class="story-intro"> For the Knights, 2017 represented a triumph of reinvention. Rising from worst to first through a gamut of new coaches, a new system and new expectations, the team banded together to ascend from the depths of a disastrous 0-12 season in 2015 to achieve a perfect record in a remarkable two years. But it wasn't a miracle. According to seniors, the incredible transformation was the result of hard work, belief and brotherhood &mdash; plus a dose of redemption. If you ask them, 2017 was "our chance to show &rsquo;em."</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="kings-court mt-7">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-10 col-md-offset-1 col-sm-10 col-sm-offset-1">
+          <span class="kings-court-header text-center" [background-image filename='Kings-Court-Header-BG']>King's Court</span>
+          <div class="mt-7">
+            <div class="col-md-4 col-xs-6">
+              <img src="[image filename='Akins-Thumbnail']" alt="Jordan Akins" class="img-circle img-responsive center-block">
+              <ul>
+                <li><strong>#88 Jordan Akins &rsquo;17</strong></li>
+                <li><strong>Position:</strong> Tight end</li>
```

Comment:
Not high-priority, but these unordered lists could be structured a little more semantically using `<dl>`'s.  Something like:
```
<strong>Player name/year</strong>
<dl>
<dt>Position:</dt>
<dd>Tight end</dd>
<dt>Hometown:</dt>
<dd>Atlanta</dd>
...
</dl>
```
We may also want to consider making the player names headings.